.github/workflows/review-code.yaml

```diff
@@ -0,0 +1,61 @@
+name: Review PR Diff
```

Comment:
Package name does not match the content of the file (ChambreDeserializer class). (Severity: Low)